__tests__/app.test.js

```diff
@@ -32,28 +33,28 @@ describe("Task-3 GET:/api/topics", () => {
       .get("/api/topics")
       .then((res) => {
         const topics = res.body.topics;
-        topics.forEach(topic => {
-          expect(topic).toHaveProperty("description",expect.any(String));
-          expect(topic).toHaveProperty("slug",expect.any(String));
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("description", expect.any(String));
+          expect(topic).toHaveProperty("slug", expect.any(String));
         });
       });
   });
 });
 
-      //  -----    4    ------
+//🚨🚨🚨🚨
+//  -----    4 - 10    ------
 describe("Task-4 GET:/api/articles", () => {
   test("respons with status 200", () => {
     return request(app).get("/api/articles").expect(200);
-  });
+  });  
   test(`An article should have the follow property:
           - propertyaurhot,
           - title,
           - article_id,
           - topic_createe_at,
           - votes,
           - article_img_url,
-          - comment_count`
-    , () => {
+          - comment_count`, () => {
     return request(app)
       .get("/api/articles")
       .then((res) => {
```

Comment:
Make sure you protect this `.forEach` with a length condition. If the array returned is unpopulated, this test will give a false positive result.